README.md

```diff
@@ -16,14 +16,14 @@ Welcome to the `movies-api` repository! This open-source project provides a comp
 
 To use our API, first, obtain a bearer token:
 
-1. Send a `GET` request to `<final URL here>/auth/token`.
+1. Send a `GET` request to `https://0kadddxyh3.execute-api.us-east-1.amazonaws.com/auth/token`.
```

Comment:
Maybe add the base URL at the top of this readme for folks. Otherwise this seems good to me.